lib/coins/ada.js

```diff
@@ -0,0 +1,41 @@
+const b58 = require('b58')
+const cbor = require('cbor-sync')
+const CRC32 = require('crc-32'); 
+
+module.exports = {depositUrl, parseUrl}
+
+function parseUrl(network, address) {
+  console.log('DEBUG16: [%s] *%s*', network, address)
+
+  // Cardano in Byron era not recognize network type (main/test) by address prefixes like f.e. bitcoin
+  if (!validate(address)) return null
```

Comment:
This is how other currencies handle nonvalid case right? No need to throw some exception? (just asking)

Reply:
Yes, this is how non-valid cases are handled.

Reply:
You are right, they fire exceptions, but for BTC this is little hidden in bs58check library